science/gerris/Portfile

```diff
@@ -1,61 +1,74 @@
-PortSystem 1.0
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
It would be better to add the modeline and fix the portfile's whitespace in a separate commit. Also, your fixed whitespace still contains tabs; please change all tabs to spaces.

Reply:
This should now be fixed.